### ISSUE TITLE
macOS VoiceOver doesn't announce when details element is expanded when using role group

### ISSUE THREAD

**Issue by selfthinker** [alphagov/reported-bugs#27]
**Upstream bug:** https://bugs.webkit.org/show_bug.cgi?id=180866
**Date:** 2017-12-15
**Reported by:** Anika Henke
**Related to:** https://github.com/alphagov/govuk_elements/issues/572

---

### Overview

VoiceOver with Safari on iOS announces the wrong state (collapsed or expanded) when the element was just collapsed. It announces the correct state when swiping forwards and back again.

### Steps to reproduce

1. Activate VoiceOver on iOS
2. Open http://jsbin.com/sasabef in Safari
3. Navigate to the details element labelled "Open this"
4. Double tap to open the element
5. Swipe forward
6. Swipe back

### Actual result

On step 4 VoiceOver says "Open this, collapsed", although the section is expanded.

### Expected result

VoiceOver should say "Open this, expanded" like it does on step 6, or like it does in macOS on step 4.

**Comment by selfthinker** [comment]
Fixed in Safari technology preview 59:
https://developer.apple.com/safari/technology-preview/release-notes/#r59
